### PR TITLE
Update dev-issue-template.md

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
Adds release note considerations callout to dev issue template

### CODE REVIEW

**Review by andrew-jameson — CHANGES_REQUESTED** [review]
Did we agree during UX sync that we'll instead bake this into the description/notes and our backlog refinement rubric for "Refined"? Would like to see our `How We Work` docs updated if so. If my recollection is right on that, then we should close out this PR.

**Comment by reitermb** [comment]
> Did we agree during UX sync that we'll instead bake this into the description/notes and our backlog refinement rubric for "Refined"? Would like to see our `How We Work` docs updated if so. If my recollection is right on that, then we should close out this PR.

We did. There's a new commit since UX sync that moves the change into the description

File: .github/ISSUE_TEMPLATE/dev-issue-template.md [hunk 1]
```diff
@@ -47,3 +49,4 @@ _Please include any questions or decisions that must be made before beginning wo
 
 - _Open Question 1_
 - _Open Question 2_
+
```
Comment (andrew-jameson) [inline .github/ISSUE_TEMPLATE/dev-issue-template.md]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/dev-issue-template.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ assignees: ''
 
 **Description:**
 _Provide a brief background and justification for this issue_
+_OFA & UX team: Please include bullets capturing the "so what?" value to STT end users, related guidance, and/or talking points that should be communicated to STTs in release notes_
+
```
Comment (andrew-jameson) [inline .github/ISSUE_TEMPLATE/dev-issue-template.md]:
```suggestion
_OFA & UX team: Please include bullets capturing the value to STT end users, related guidance, and/or talking points that should be communicated to STTs in release notes_
```

**Review by andrew-jameson — CHANGES_REQUESTED** [review]
Feedback on new commit.

**Review by andrew-jameson — APPROVED** [review]

**Review by lfrohlich — APPROVED** [review]

**Review by ADPennington — APPROVED** [review]
👍🏾

**Pull request merged**